ingest/ingest_pipeline.py

```diff
@@ -537,6 +540,11 @@ def extract_from_anndata(self):
                 "extract"
             ):
                 self.anndata.generate_processed_matrix(self.anndata.adata)
+
+            if self.kwargs.get('extract') and "raw_counts" in self.kwargs.get(
+                "extract"
+            ):
```

Comment:
Tidier:

```suggestion
            if self.kwargs.get('extract') and "raw_counts" in self.kwargs.get('extract'):
```

or, less ideally but smoother than current IMO:

```
if (
    self.kwargs.get('extract') and 
    "raw_counts" in self.kwargs.get('extract')
):
```

Alas, [Python lacks elegant optional chaining](https://stackoverflow.com/questions/64285182/optional-chaining-for-python-objects-foo-bar-baz).

Reply:
Agreed - most of this was me trying to match syntax of what we had before.